test/unit/host_test.rb

```diff
@@ -2719,6 +2719,23 @@ def to_managed!
     end
   end
 
+  context 'hostgroup defaults' do
+    setup do
+      @group1 = FactoryGirl.create(:hostgroup, :compute_profile => compute_profiles(:one))
+      @group2 = FactoryGirl.create(:hostgroup, :compute_profile => compute_profiles(:two))
+    end
+
+    test 'sets proper compute attributes on hostgroup change' do
+      host = FactoryGirl.create(:host, :managed, :compute_resource => compute_resources(:one), :hostgroup => @group1)
+      host.set_hostgroup_defaults
```

Comment:
It's not exactly difference, I'm testing that compute attributes have actually changed to the second compute profile. I just want to make sure that the setup for the test does not make the expectation true, even if the attributes did not change.